### PR TITLE
Automattic for Agencies: Update WP.com auth pages

### PR DESCRIPTION
**Pull request by yashwin**
Resolves https://github.com/Automattic/jetpack-genesis/issues/217

## Proposed Changes

- Adds logo for Automattic for Agencies
- Adds logo for A4A + WP 
- Updates signup page for A4A
- Updates login page for A4A

## Testing Instructions

Note: We will update the theme colors in another PR.

1) Open the Calypso live link
2) Append the URL with `/log-in?client_id=95928&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95928%26redirect_uri%3Dhttps%253A%252F%252Fagencies.localhost%253A3000%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1`. `95928` is the dev OAuth client ID and verify that the page is updated as shown below:

Screenshot 2024-02-15 at 10 10 18 AM

3) Click the `Create an account` button -> Verify that the page is updated as shown below:

Screenshot 2024-02-13 at 8 23 25 PM

4) Repeat step 2 with OAuth client ID: Jetpack `/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1`. Verify that the page has Jetpack Logo.

5. Repeat step 3 and verify that the Jetpack logo is shown

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Comment by jeffgolenski** [comment]
Combo logo with the + symbol looks good @yashwin. 🚢

**Review by oikeme — APPROVED** [review]
Additions look good and changes for A4A work as expected :shipit:

File: client/lib/login/index.js [hunk 1]
```diff
@@ -121,14 +122,6 @@ export function getSignupUrl( currentQuery, currentRoute, oauth2Client, locale,
 		return `${ signupUrl }/wpcc?${ oauth2Params.toString() }`;
 	}
 
-	if ( oauth2Client && isJetpackCloudOAuth2Client( oauth2Client ) ) {
```
Comment (jkguidaven) [inline client/lib/login/index.js:124]: I'm curious why these codes are removed.

**Review by jkguidaven — APPROVED** [review]
tested the changes it works as expected. 👍 LGTM

File: client/lib/login/index.js (hunk 1 above)
Comment (yashwin) [reply client/lib/login/index.js:124]: It was basically doing the same thing: 

Screenshot 2024-02-15 at 3 05 57 PM

**Comment by yashwin** [comment]
Thanks for the review, @oikeme & @jkguidaven! 🙇

File: client/lib/login/index.js (hunk 1 above)
Comment (jkguidaven) [reply client/lib/login/index.js:124]: Ahh thanks for the clarifications.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11349155

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11349155).

**Pull request merged**